server/plugins/modelPlugin.js

```diff
@@ -265,6 +266,9 @@ class ModelPlugin {
         } catch (error) {
             // Log the error and continue
             console.error(error);
+            if(responseData?.error?.response?.status == 400){
+                throw error;
```

Comment:
Should we send a message back in all cases? Seems like in this case, it'll still silently fail if the error is non-400, like 422 or something.

Reply:
updated to throw all at plugin level